### PR TITLE
Feat/account page

### PR DESCRIPTION
**Pull request by ezeaniiandrew**
I'm done with the user's account page

### CODE REVIEW

**Review by bibhestee — APPROVED** [review]

**Review by Hemazyn — COMMENTED** [review]
good.

File: src/components/Avatar/avatar.module.css
Comment (Leeyah-123) [inline src/components/Avatar/avatar.module.css:1]: We're using tailwindcss. Please migrate the css styles to tailwind CSS.

File: src/components/EventCard/event-card.module.css
Comment (Leeyah-123) [inline src/components/EventCard/event-card.module.css:1]: Migrate to tailwind

**Review by Hemazyn — APPROVED** [review]
Good.

**Review by Leeyah-123 — CHANGES_REQUESTED** [review]
Please migrate all CSS styles to tailwind CSS and remove the CSS files.

File: src/components/Avatar/avatar.module.css
Comment (ezeaniiandrew) [reply src/components/Avatar/avatar.module.css:1]: I don't know how to use tailwind. Trying to learn it on this project will not be possible due to time constraint.

Comment (bibhestee) [reply src/components/Avatar/avatar.module.css:1]: Migrating to tailwind might take time, he can start using tailwind on the next task.

File: src/components/TabPanel/tab-panel.module.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.panel {
+  margin-block-start: 1rem;
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(260px, 1fr));
+  gap: 1rem;
+}
+
+.hide {
```
Comment (Leeyah-123) [inline src/components/TabPanel/tab-panel.module.css]: Rename this to "hidden" instead

File: src/components/EventCard/index.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { faBookmark } from "@fortawesome/free-solid-svg-icons";
+import { useState } from "react";
+import { faBookmark as notBookmarked } from "@fortawesome/free-regular-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import styles from "./event-card.module.css"
+
+const imageUrl = `https://s3-alpha-sig.figma.com/img/699d/1183/8d9716e61a45f9e95043b56334473c19?Expires=1714348800&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=RZ4xRYD3OvMQM-9CYNL9jNLYbACRF-T7yDNBW904td9Y3hw6WnzRARE8iebdaW0X930VG0s~eD2FQ1h4DCy2Oyly9ajQBeGkpdN1XR2zXRCaVt~n7NxYtdEORMyqYq18Q6q1a6jEUGdnxBtA2Z22UaPKHK4BxrifO3PYCqr6MTLMnmggWE3VnSVBu751JDB2WwyUKbV~vLOmFdoeeWENxxirrIP-3d6agV5darB9BgxV31VawN6DNPJ8YKYSg4vURbetFmbTJCxmHkkbq8TnfKgUwxs3x0aLmo8Op1BtBLNcz7rPc8fgZGQZFBkQS~6U8hZ1n3E1w66OIELE4Mn7kA__`;
```
Comment (Leeyah-123) [inline src/components/EventCard/index.jsx:7]: Export the image from figma as PNG or SVG, add it to the codespace and use the relative url.

Comment (ezeaniiandrew) [reply src/components/EventCard/index.jsx:7]: It's a placeholder image. It would be removed once the page is functional

**Comment by Leeyah-123** [comment]
Is this how this should look?

image

File: src/components/TabPanel/tab-panel.module.css (hunk 1 above)
Comment (ezeaniiandrew) [reply src/components/TabPanel/tab-panel.module.css]: Done

**Comment by ezeaniiandrew** [comment]
> Is this how this should look?
> 
> image

I only built the user's account page. I don't know who built this page

File: src/components/EventCard/event-card.module.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
+.card {
```
Comment (Leeyah-123) [inline src/components/EventCard/event-card.module.css:1]: The border radius on the card does not match that on the figma.

File: src/components/EventCard/index.jsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+                onClick={() => setIsBookmarked(!isBookmarked)}
+            >
+                {isBookmarked ? (
+                    <FontAwesomeIcon icon={faBookmark} />
+                ) : (
+                    <FontAwesomeIcon icon={notBookmarked} />
+                )}
+            </div>
+            <img src={imageUrl} alt="" />
+            <div className={styles.details}>
```
Comment (Leeyah-123) [inline src/components/EventCard/index.jsx:25]: In the design, the background image does not cover the event details. Please update that.

**Review by Leeyah-123 — APPROVED** [review]

**Pull request merged**